### PR TITLE
split video_insights stream into lifetime and daily

### PR DESCRIPTION
**Pull request by yujoy**

### CODE REVIEW

File: tap_facebook_pages/client.py [hunk 1]
```diff
@@ -143,6 +143,14 @@ def validate_response(self, response: requests.Response) -> None:
                 == "(#190) This method must be called with a Page Access Token"
             ):
                 raise RetriableAPIError(msg)
+            monetization_access_pattern = re.compile(
+                "^.*Monetization metrics are only visible for Page admins.*$"
+            )
+            if response.status_code == 403 and monetization_access_pattern.match(
+                response.json().get("error", {}).get("message")
+            ):
+                self.logger.warning(f"Skipping record because: {msg}")
```
Comment (prratek) [inline tap_facebook_pages/client.py:152]: Just checking - we don't have access to the video ID here right? Could be useful to log that if we did, but I'm noticing I didn't do that for the post insights stream validation a few lines above

File: tap_facebook_pages/streams.py [hunk 2]
```diff
@@ -687,10 +687,10 @@ def parse_response(self, response: requests.Response) -> Iterable[dict]:
 #         # "post_video_ad_break_ad_cpm",
 #     ]
 
-class VideoInsightsStream(InsightsStream):
+class VideoInsightsLifetimeStream(InsightsStream):
```
Comment (prratek) [inline tap_facebook_pages/streams.py:690]: Should we explicitly set the period to `lifetime`? Just want to make sure we defend against FB randomly deciding to also return daily values here in the future

**Review by prratek — COMMENTED** [review]
added a couple of minor comments

File: tap_facebook_pages/client.py (hunk 1 above)
Comment (yujoy) [reply tap_facebook_pages/client.py:152]: Yeah doesn't seem it like, response looks like this - 

Screen Shot 2022-12-28 at 3 25 29 PM

**Pull request merged**